Apps/W1/Shopify/app/src/Order handling/Codeunits/ShpfyProcessOrder.Codeunit.al

```diff
@@ -121,6 +122,10 @@ codeunit 30166 "Shpfy Process Order"
                 SalesHeader.Validate("Tax Area Code", ShopifyTaxArea."Tax Area Code");
             if ShopifyOrderHeader."Shipping Method Code" <> '' then
                 SalesHeader.Validate("Shipment Method Code", ShopifyOrderHeader."Shipping Method Code");
+            if ShopifyOrderHeader."Shipping Agent Code" <> '' then begin
+                SalesHeader.Validate("Shipping Agent Code", ShopifyOrderHeader."Shipping Agent Code");
+                SalesHeader.Validate("Shipping Agent Service Code", ShopifyOrderHeader."Shipping Agent Service Code");
+            end;
```

Comment:
It would be nice if can have some unit tests on this. I know we are lacking on shipping charges tests but we have examples on "Tax Area Code".

Examples in codeunit 139608, UnitTestCreateSalesDocumentTaxPriorityCode

Reply:
@aidasberesinevicius fyi

Reply:
I would think any development task should include unit tests by default in today's world. How would you even know if your implementation is covering all scenarios and working properly... Nevertheless, I will resolve this and merge it and keep this in mind for future projects.